src/moveit_commander/robot.py

```diff
@@ -177,12 +263,15 @@ def get_default_owner_group(self, joint_name):
         return self._joint_owner_groups[joint_name]
 
     def __getattr__(self, name):
-        """ We catch the names of groups, joints and links to allow easy access to their properties """
+        """
+        We catch the names of groups, joints and links to allow easy access
+        to their properties.
+        """
         if name in self.get_group_names():
             return self.get_group(name)
         elif name in self.get_joint_names():
-            return self.Joint(self,name)
+            return self.Joint(self, name)
         elif name in self.get_link_names():
-            return self.Link(self,name)
+            return self.Link(self, name)
```

Comment:
is removing the space between variables an improvement for python style? i'm not a python expert. i see on line 265 (of the modified file) there is a space that was not removed:

    def __getattr__(self, name):

seems inconsitent